crypto/sha/keccak1600.c

```diff
@@ -11,6 +11,10 @@
 #include <string.h>
 #include <assert.h>
 
+size_t SHA3_absorb(uint64_t A[5][5], const unsigned char *inp, size_t len,
+                   size_t r);
+void SHA3_squeeze(uint64_t A[5][5], unsigned char *out, size_t len, size_t r);
+
```

Comment:
To produce pairs of binaries with -DSELFTEST that I can single-step in debugger side by side and compare. I mean in order to do that I have to type different compiler commands, different depending on what I'm looking at at any particular moment.